packages/react-union-scripts/package.json

```diff
@@ -21,12 +21,12 @@
     "babel-loader": "^8.0.2",
     "babel-preset-react-union": "^0.7.1",
     "browser-sync": "2.23.6",
-    "bundle-loader": "0.5.6",
```

Comment:
you will need to add upgrade guide

Reply:
Will do

Reply:
https://github.com/lundegaard/react-union.org/issues/21 issue created here